avsomr/avsimport.cpp

```diff
@@ -84,14 +84,15 @@ static Score::FileError doImportMSMR(MasterScore* score
                   return err;
 
             // corrections after import
+           QString titleText = baseName;
             Text* title = score->getText(Tid::TITLE);
             if (title)
-                  title->setPlainText(baseName);
+                  title->setPlainText(titleText);
```

Comment:
? Rather `if (!title)`

Reply:
sorry, did not understand the comment

Reply:
It title is not set

Reply:
If it is set, why overwrite it?

Reply:
Because Audiveris sets the title "[Audiveris detected movement]", I decided it was not convenient for the user, the file name title is more convenient.

Reply:
added comment in the code